pumpkin-lib/src/engine/constraint_satisfaction_solver.rs

```diff
@@ -395,6 +401,13 @@ impl ConstraintSatisfactionSolver {
         proof.optimal(bound, &self.variable_names, &self.variable_literal_mappings)
     }
 
+    fn complete_proof(&mut self) {
+        pumpkin_assert_simple!(self.is_conflicting());
+
+        let _ = self.compute_learned_clause(&mut DummyBrancher);
+        let _ = self.internal_parameters.proof_log.log_learned_clause([]);
```

Comment:
Wouldn't logging the output of `compute_learned_clause` make more sense here? (This way it would automatically check if something goes wrong with the conflict analysis)